Sources/Examples/HelloSwift.swift

```diff
@@ -4,52 +4,63 @@ import SwiftWin32
 
 class SwiftApplicationDelegate: ApplicationDelegate {
   var window: Window?
-
-  func application(_: Application,
-                   didFinishLaunchingWithOptions options: [Application.LaunchOptionsKey:Any]?) -> Bool {
-    let SwiftWindowClass: WindowClass =
-    WindowClass(hInst: GetModuleHandleW(nil), name: "Swift.Window")
-    _ = SwiftWindowClass.register()
-
-    class SwiftWindowDelegate: WindowDelegate {
-      func OnClose(hWnd: HWND, _: WPARAM, _: LPARAM) -> LRESULT {
-        DestroyWindow(hWnd)
-        return 0
+  var btn: Button?
+  var label: Label?
+  
+  class MyBtnDelegate: ViewDelegate {
+    var view: View?
+    var red: Bool = true
+    
+    public func OnPaint(clientRect: RECT, hdc: HDC?) -> LRESULT {
+      if let view = self.view {
+        let btn = view as! Button
+        let bg = btn.backgroundColor
+        let fg = btn.textColor
+        
+        PaintHelpers.DrawBackground(clientRect: clientRect, hdc: hdc, color: bg)
+        PaintHelpers.DrawText(clientRect: clientRect, hdc: hdc, text: btn.text, color: fg)
       }
-
-      func OnDestroy(_: HWND, _: WPARAM, _: LPARAM) -> LRESULT {
-        PostQuitMessage(0)
-        return 0
-      }
-
-      func OnPaint(_ hWnd: HWND, _: WPARAM, _: LPARAM) -> LRESULT {
-        let red: Color = #colorLiteral(red: 1.0, green: 0.0, blue: 0.0, alpha: 1.0)
-
-        var rctRect: RECT = RECT()
-
-        GetClientRect(hWnd, &rctRect)
-
-        var psPaint: PAINTSTRUCT = PAINTSTRUCT()
-        let hDC: HDC = BeginPaint(hWnd, &psPaint)
-          SelectObject(hDC, GetStockObject(NULL_PEN))
-          SelectObject(hDC, GetStockObject(DC_BRUSH))
-          SetDCBrushColor(hDC, red.COLORREF)
-          Rectangle(hDC, rctRect.left, rctRect.top, rctRect.right, rctRect.bottom)
-        EndPaint(hWnd, &psPaint)
-
-        return 0
+      return 0
+    }
+      
+    func OnClick(pos: Point) -> LRESULT {
+      print("CLICK WINDOW")
+      red = !red
+      if red {
+        self.view?.backgroundColor = #colorLiteral(red: 0.0, green: 0.0, blue: 1.0, alpha: 1.0)
+      } else {
+        self.view?.backgroundColor = #colorLiteral(red: 1.0, green: 0.0, blue: 0.0, alpha: 1.0)
       }
+      return 0
     }
+  }
+  
+  func application(_: Application,
+                   didFinishLaunchingWithOptions options: [Application.LaunchOptionsKey:Any]?) -> Bool {
+      
+    self.window = Window(title: "Swift Window")
+    self.btn = Button(title: "Button")
+    self.label = Label(title: "Label")
 
-    self.window = Window(class: SwiftWindowClass, title: "Swift Window")
-    self.window?.delegate = SwiftWindowDelegate()
-    self.window?.show()
+    self.btn?.backgroundColor = #colorLiteral(red: 0.0, green: 0.0, blue: 1.0, alpha: 1.0)
+    
+    self.btn?.text = "BUTTON HI"
+    self.label?.text = "LABEL HI"
+    self.label?.textColor = #colorLiteral(red: 0.0, green: 1.0, blue: 0.0, alpha: 1.0)
+    self.label?.borderThickness = 10
 
+    self.window?.addSubview(self.btn ?? Button(title: ""))
```

Comment:
I think it would be more concise to unwrap the button and label here instead of using `??`. (Perhaps we could also store the unwrapped values to variables instead of using `self.btn?` everywhere?)
```suggestion
    self.window?.addSubview(self.btn!)
```